### DISCUSSION TITLE
Only Flash Bootloader Using Initrd

### DISCUSSION THREAD

**Discussion by CHNtentes** [OE4T/meta-tegra#1207] [Q&A]
Hi. I'd like to flash only the bootloader into Orin NX using NVMe. I saw [https://github.com/OE4T/meta-tegra/discussions/728](url), but initrd-flash does not accept "-c" option. What command should I use?

**Comment by dwalkes** [comment]
I don't think the initrd-flash script currently supports this, you'd need to add this using https://github.com/OE4T/meta-tegra/blob/master/recipes-bsp/tegra-binaries/tegra-helper-scripts/initrd-flash.sh

Bootloader updates should also be possible through UEFI capsule partition updates.  Work for this is in progress, see https://matrix.to/#/#OE4T_jetpack-5-device-update:gitter.im if you'd like to follow the work there or help.
